pkg/proxy/hybrid/proxy.go

```diff
@@ -24,7 +25,8 @@ type HybridProxier struct {
 
 	// TODO(directxman12): figure out a good way to avoid duplicating this information
 	// (it's saved in the individual proxies as well)
-	usingUserspace map[types.NamespacedName]struct{}
+	usingUserspace map[types.NamespacedName]bool
```

Comment:
please leave this as `struct{}` -- it's the recommended method doing sets (it takes up no space for the value).